packages/banner/ember-backstop/backstop.js

```diff
@@ -0,0 +1,38 @@
+const path = require('path');
```

Comment:
Can we keep all the backstop.js files in one place and symlink it in individual packages, seems like it is repeating?